### PR TITLE
Update jquery and flot versions

### PR DESCRIPTION
**Pull request by DomGarguilo**
This PR updates
* jquery from 3.2.1 to 3.6.1
* flot from 4.2.1 to 4.2.3

For jquery, I used the "Download the uncompressed, development jQuery 3.6.1" link from https://jquery.com/download/
For flot, I downloaded the files from https://github.com/flot/flot/releases/tag/v4.2.3

I updated the LICENSE files in our repo but noticed the following:
https://github.com/apache/accumulo/blob/d51e2f1c4455012b66d38600a3579b5d233fc4e3/LICENSE#L298-L300

but don't really know what that is referring to (maybe a previous version of [this](https://github.com/flot/flot/blob/efb67df19a4cf70316d4fca7ed2a86e73e6b4181/source/jquery.flot.pie.js)?). And also found this in flot so im not too sure if it should be remove from our licensing or not as it might be left over from an old version of flot that we were using:

> <li>The pie plug-in is now part of the Flot repository! This should make it a lot easier to deal with.</li>

https://github.com/flot/flot/blob/d0f569a47566c61ef7a43e94e5daaacd1a18bf4e/examples/series-pie/index.html#L768

### CODE REVIEW

**Review by cshannon — COMMENTED** [review]
I reviewed and tested this and the changes LGTM. However, one question I had is why are we using the uncompressed jQuery version? I would think we'd want to distribute the compressed version as uncompressed is meant for development use.

For the licensing issue, I'm not sure so I'll let @ctubbsii or someone else comment on that.

**Comment by ctubbsii** [comment]
@DomGarguilo Regarding the license issue, flot used to bundle some jQuery plugins, with their own license specifics. If those are no longer bundled in the version of flot we're using (and therefore, we are no longer redistributing them), we can get rid of those mentions in the license.

@cshannon We've discussed this before, but I'm too lazy to dig it up in the archives at the moment. It boils down to: obfuscated/minimized code is not free and open source. It does not meet the definition of open source software according to the the open source initiative (not ASF's official definition, but a useful one), nor does it meet the definition of free software according to the free software foundation (again, not ASF's official definition, but a useful one). We should not bundle the minimized versions in our source tarball. We could include them in our distribution tarball, but there's really no reason to do that, because we're probably not getting much benefit from it anyway. Many browsers cache and optimize and do other various magical things with javascript... and that's assuming downstream users haven't modified the monitor to use a CDN instead of the javascript we're giving them anyway. So, using the obfuscated/minimized versions are generally not worth it for us.

**Comment by cshannon** [comment]
I did some digging with the licensing and it looks jquery.pie.js is under https://github.com/flot/flot/blob/master/source/jquery.flot.pie.js. The licensing in the headers has been renamed/reworked slightly but it is still used.

The section that we currently show in our license file referring to June 2009 creation was dropped long ago:
 
https://github.com/flot/flot/commit/9e21074d82376c519ca586b1fdcfca0eafdc97ad#diff-46d28d77e2699e7634fe9138e185409139794584ffa0c7cfb842c79efcaa8741

Current version shows:

https://github.com/flot/flot/blob/de34ce947d8ebfb2cac0b682a130ba079d8d654b/source/jquery.flot.pie.js#L1-L14

@ctubbsii  - Should we just update the LICENSE file with the updated wording from the current version?

As a side note, I also checked the licensing section on jquery.flot.resize.js and it is still accurate with the inline dependency. I think only the jquery.pie.js part should be removed.

**Comment by ctubbsii** [comment]
> Should we just update the LICENSE file with the updated wording from the current version?

Yes, exactly.

**Comment by DomGarguilo** [comment]
Thanks for the input @ctubbsii and @cshannon. I updated the flot license. Take a look and see what you think.

File: LICENSE [hunk 1]
```diff
@@ -295,9 +295,21 @@ Files:
...
+
+        Copyright (c) 2007-2014 IOLA and Ole Laursen.
+        Licensed under the MIT license.
+
+        The plugin assumes that each series has a single data value, and that each
+        value is a positive integer or zero.  Negative numbers don't make sense for a
+        pie chart, and have unpredictable results.  The values do NOT need to be
+        passed in as percentages; the plugin will calculate the total and per-slice
+        percentages internally.
+
```
Comment (ctubbsii) [inline LICENSE:309]: This is not part of the license/attribution content. It's a description of the plugin, which does not need to be included here. It is unfortunate they intermingled the description with the attribution notices. I would snip out the description from the middle of this in our files.

File: assemble/src/main/resources/LICENSE [hunk 2]
```diff
@@ -294,9 +294,21 @@ Files:
...
+
+        Copyright (c) 2007-2014 IOLA and Ole Laursen.
+        Licensed under the MIT license.
+
+        The plugin assumes that each series has a single data value, and that each
+        value is a positive integer or zero.  Negative numbers don't make sense for a
+        pie chart, and have unpredictable results.  The values do NOT need to be
+        passed in as percentages; the plugin will calculate the total and per-slice
+        percentages internally.
+
```
Comment (ctubbsii) [inline assemble/src/main/resources/LICENSE:308]: Same comment as above

File: server/monitor/src/main/appended-resources/META-INF/LICENSE [hunk 3]
```diff
@@ -57,9 +57,21 @@ Files:
...
+
+        Copyright (c) 2007-2014 IOLA and Ole Laursen.
+        Licensed under the MIT license.
+
+        The plugin assumes that each series has a single data value, and that each
+        value is a positive integer or zero.  Negative numbers don't make sense for a
+        pie chart, and have unpredictable results.  The values do NOT need to be
+        passed in as percentages; the plugin will calculate the total and per-slice
+        percentages internally.
+
```
Comment (ctubbsii) [inline server/monitor/src/main/appended-resources/META-INF/LICENSE:71]: Same comment as above

File: server/monitor/src/main/appended-resources/META-INF/LICENSE [hunk 4]
```diff
@@ -57,9 +57,21 @@ Files:
...
+        * Created by Brian Medendorp
+
```
Comment (ctubbsii) [inline server/monitor/src/main/appended-resources/META-INF/LICENSE:73]: Can remove this blank line (same for the above copies) between these attributions.

**Review by ctubbsii — COMMENTED** [review]
I know this already got merged, but I think it should be further tweaked in very minor ways.

**Pull request merged**